crates/frontend/src/components/experiment.rs

```diff
@@ -264,6 +262,64 @@ where
 
                 </div>
             </div>
+            <Show when=move || {
+                experiment
+                    .with_value(|v| {
+                        v.metrics.enabled || matches!(v.status, ExperimentStatusType::CREATED)
```

Comment:
```suggestion
                        v.metrics.enabled || v.status == ExperimentStatusType::CREATED
```
Just to avoid the macro